service/controller/internal/nodecontroller/resource/readylabel/create.go

```diff
@@ -0,0 +1,45 @@
+package readylabel
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/giantswarm/kvm-operator/service/controller/key"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
```

Comment:
This is the only resource in the dynamic controller. It manages an annotation to MC node pods reflecting the ready status of corresponding WC nodes.